examples/guestbook/README.md

```diff
@@ -426,39 +530,56 @@ spec:
 
 ```console
 $ kubectl get rc
-CONTROLLER                             CONTAINER(S)            IMAGE(S)                                   SELECTOR                     REPLICAS
-frontend                               php-redis               kubernetes/example-guestbook-php-redis:v3  name=frontend                3
-redis-master                           master                  redis                                      name=redis-master            1
-redis-slave                            slave                   gcr.io/google_samples/gb-redisslave:v1     name=redis-slave             2
+CONTROLLER                  CONTAINER(S)            IMAGE(S)                                   SELECTOR                               REPLICAS
+frontend                    php-redis               kubernetes/example-guestbook-php-redis:v3  app=guestbook,tier=frontend            3
+redis-master                master                  redis                                      app=redis,role=master,tier=backend     1
+redis-slave                 slave                   gcr.io/google_samples/gb-redisslave:v1     app=redis,role=slave,tier=backend      2
 ```
 
-Once it's up (again, it may take up to thirty seconds to create the pods) you can list the pods in the cluster, to verify that the master, slaves and frontends are all running.  You should see a list that includes something like the following:
+Once it's up (again, it may take up to thirty seconds to create the pods) you can list the pods with specified labels the cluster, to verify that the master, slaves and frontends are all running. You should see a list contains pods with label tier like the following:
 
 ```console
-$ kubectl get pods
-NAME                                           READY     STATUS    RESTARTS   AGE
-...
-frontend-4o11g                                 1/1       Running   0          2h
-frontend-u9aq6                                 1/1       Running   0          2h
-frontend-yga1l                                 1/1       Running   0          2h
-...
-redis-master-dz33o                             1/1       Running   0          2h
-redis-slave-35mer                              1/1       Running   0          2h
-redis-slave-iqkhy                              1/1       Running   0          2h
+$ kubectl get pods -L tier
+NAME                                READY     STATUS    RESTARTS   AGE     TIER
+frontend-4o11g                      1/1       Running   0          2h      frontend
+frontend-u9aq6                      1/1       Running   0          2h      frontend
+frontend-yga1l                      1/1       Running   0          2h      frontend
+redis-master-dz33o                  1/1       Running   0          2h      backend
+redis-slave-35mer                   1/1       Running   0          2h      backend
+redis-slave-iqkhy                   1/1       Running   0          2h      backend
 ```
 
 You should see a single redis master pod, two redis slaves, and three frontend pods.
```

Comment:
Can you change L587 from "guestbook/php-redis/guestbook.php" to "examples/guestbook/php-redis/guestbook.php"